api/src/services/users/users.js

```diff
@@ -3,14 +3,14 @@ import { db } from 'src/lib/db'
 // Only to be used on the api side
 export const getCustomerId = async ({ id }) => {
   return await db.user.findUnique({
-    where: { id: parseInt(id) },
+    where: { id: id },
   })
 }
 
-export const getUserByCustomerId = ({ customerId }) => {
+export const getUserByCustomerId = ({ id }) => {
   return db.user.findUnique({
     where: {
-      customerId: customerId,
+      id: id,
```

Comment:
If they're named the same you can do:
```suggestion
      id,
```